apps/teams-test-app/src/components/DialogAPIs.tsx

```diff
@@ -25,6 +36,35 @@ const DialogAPIs = (): ReactElement => {
     setResizeRes('Teams client SDK call dialog.resize was called');
   };
 
+  const sendMessageToChild = (message: string): void => {
+    if (childWindowRef.current && childWindowRef.current !== null) {
+      const childWindow = childWindowRef.current as IAppWindow;
+      childWindow.postMessage(message).then(() => {
+        setSendMessageToChildRes('Message sent to child');
+      });
+    } else {
+      setSendMessageToChildRes('childWindow doesnt exist');
+    }
+  };
+
+  const sendMessageToParent = (message: string): void => {
+    const parentWindow = ParentAppWindow.Instance;
+    if (parentWindow) {
+      parentWindow.postMessage(message).then(() => {
+        setSendMessageToParentRes('Message sent to parent');
+      });
+    } else {
+      setSendMessageToParentRes('parentWindow doesnt exist');
+    }
+  };
+
+  const registerForParentMessage = (): void => {
```

Comment:
Do we need a similar one for `registerForChildMessage`?

Reply:
On dialog.open click we auto register for child message and its text box displays messages from child